profile/nodejs.sh

```diff
@@ -30,11 +42,9 @@ export PATH="$HOME/.heroku/node/bin:$HOME/.heroku/yarn/bin:$PATH:$HOME/bin:$HOME
 export NODE_HOME="$HOME/.heroku/node"
 export NODE_ENV=${NODE_ENV:-production}
 
-calculate_concurrency
-
-export MEMORY_AVAILABLE=$MEMORY_AVAILABLE
-export WEB_MEMORY=$WEB_MEMORY
-export WEB_CONCURRENCY=$WEB_CONCURRENCY
+export MEMORY_AVAILABLE=${MEMORY_AVAILABLE-$(detect_memory 512)}
+export WEB_MEMORY=${WEB_MEMORY-512}
+export WEB_CONCURRENCY=$(calculate_concurrency $MEMORY_AVAILABLE $WEB_MEMORY)
```

Comment:
instead of the `-z` check in `calculate_concurrency()` you could also do `export WEB_CONCURRENCY=${WEB_CONCURRENCY:-$(calculate_concurrency $MEMORY_AVAILABLE $WEB_MEMORY)}`